src/mmds/src/token.rs

```diff
@@ -0,0 +1,563 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
I'd like to have the implementation details documented in: https://github.com/firecracker-microvm/firecracker/blob/main/docs/mmds/mmds-design.md.

Will this come in a future PR?

Reply:
Yes, documentation updates regarding the implementation, as well as usage and examples will come in a future PR.